### DISCUSSION TITLE
Method for the Thermal Conductance Calculation

### DISCUSSION THREAD

**Discussion by ksauni** [ttadano/alamode#177] [Q&A]
What method does ALAMODE use to compute the thermal conductivity under different temperature? Is it non-equilibrium green function (NEGF) method or non-equilibirum molecular dynamics (NEMD)?

Does ALAMODE consider the phonon-phonon interaction and electron-phonon interaction for the thermal conductivity calculation?

Would anyone please give some explanation?

Thank you in advance.

**Comment by ttadano** [comment]
ALAMODE computes the thermal conductivity based on Boltzmann transport theory in the diffusive regime. It does not support NEGF nor NEMD.
